src/kOS/Communication/InterVesselManager.cs

```diff
@@ -0,0 +1,87 @@
+﻿using System;
+using System.Collections.Generic;
+using kOS.Safe.Encapsulation;
+using kOS.Safe.Serialization;
+using kOS.Serialization;
+using kOS.Safe;
+
+namespace kOS.Communication
+{
+    [KSPScenario(ScenarioCreationOptions.AddToAllGames | ScenarioCreationOptions.AddToExistingGames, GameScenes.FLIGHT, GameScenes.SPACECENTER, GameScenes.TRACKSTATION)]
```

Comment:
I've looked at some different mod and have stolen that piece of code.

You're right: I need one instance and load/save capability. I don't know how else I can achieve that. Feel free to change this part if you find something better.